k4EDM4hep2LcioConv/src/k4Lcio2EDM4hepConv.cpp

```diff
@@ -288,15 +260,65 @@ namespace LCIO2EDM4hepConv {
     return dest;
   }
 
+  std::vector<CollNamePair> convertLCIntVec(const std::string &name, EVENT::LCCollection *LCCollection) {
+    // Since podio doesnt have a structure for vector of vector of int or
+    // a collection that can be filled with multiple vector<int>,
+    // a workaround had to be found to convert the LCIO data.
+    // All the data gets put into one Collection with an additional Collection
+    // holding the beginnings and ends of the original vectors.
```

Comment:
This needs to go to the docstring in the header, since it is important information that should not be hidden in the implementation.